Assets/Scripts/HealthScript.cs

```diff
@@ -27,6 +27,7 @@ public HealthScript(float maxHealth)
 
     private void Start()
     {
+        // TODO: Race condition here with this and the individual creature.Start() commands
```

Comment:
🤔 can you make one script depend on another and that "solves" it?

https://discussions.unity.com/t/unit-scripts-calling-start-function-before-other-scripts/851480/4

interesting ideas here, one of making something a coroutine? I remember using that for something... I think the effectsmanager?

Reply:
https://trello.com/c/i7Nvp2HQ/95-resolve-race-condition-between-healthscript-and-creaturescripts